tic-tac-toe/src/components/EndScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import "../styles/EndScreen.css";
+import { Winner } from "../utils";
+import { colors } from "../constants";
+import caelin from "../images/Caelin.png";
+
+export interface Props {
+  winner: Winner;
+  resetScore: () => void;
+  playAgain: () => void;
+}
+
+const EndScreen = (props: Props) => {
+  let text;
+  if (props.winner !== "DRAW") {
```

Comment:
This is very inefficient, will run this logic on each render. Do this inline in the rendering function rather than having this variable